docs/install/install-cave.md

```diff
@@ -51,6 +51,12 @@ Regardless of what Operating System CAVE is running on, these general requiremen
 
 !!! note "While CentOS8 has reach End of Life as of Dec. 31, 2021, CentOS7 End of Life isn't until June 30, 2024."
 
+### Upgrade Existing Installation
+
+If you do not currently have CAVE installed, please go directly to the [Download and Installation Instructions](#download-and-installation-instructions).
+
+If you already have CAVE installed, following the [Download and Installation Instructions](#download-and-installation-instructions) below will also properly take care of your existing install.
+
```

Comment:
I did actually have that in there...but then was wondering if we should do it for every OS then (have that sentence linking to the proper section at the bottom), and then I opted just to remove it instead of including it in all the sections?

Reply:
Yeah...Linux is different than Windows and Mac so I think the link to fully remove is needed for sure here

Reply:
Ok sounds good 👍